### ISSUE TITLE
Logos are not loading on Nextcloud 15

### ISSUE THREAD

**Issue by apinter** [mwalbeck/nextcloud-breeze-dark#55]
Hi,
   
Not sure if I should ask here or on the NC forum so sorry in advance if my question supposed to be posted there. I use nextcloud-breeze-dark since NC14 and love it. The only thing I used to change is the logo on the login page, the logo in the upper left corner after logged in and the favicon. After upgrading to NC15 I noticed that these logos are not loading anymore. The logos included in nextcloud-breeze-dark are not loaded either instead it is loading the original white logos of NC. However enabling the Theming app will load the logos and also keep the changes made by applying this theme. Is this the new normal and can't load custom logos any longer via theming and need to use the app or I missed something?

EDIT: Turns out that I'm a muppet and - surprise, surprise - adding the .logo and .logo-icon headers solves the "problem". Sorry, I'm gonna go now....

**Comment by mwalbeck** [comment]
The situation is they made some changes for the Theming app in NC15 which made it possible to use with a custom theme. Since I've had a few request to be able to use the theming app with the theme I decided to make the necessary changes. That  meant going back to the default Nextcloud logo as the theme CSS is loaded after the Theming app so any changes to the logo would just be overwritten by the theming app. So they haven't limited the options for changing the logos but I have made the Theming app the default option for this theme as it is the most accessible option. Hope that clarifies things.
